### PR TITLE
fix(ui5-link): rename wrap property to wrappingType

### PR DESCRIPTION
**Pull request by gmkv**
Part of #3107

- fixed documentation errors

BREAKING_CHANGE: property wrap has been renamed to wrappingType and its type has changed

### CODE REVIEW

**Comment by tsanislavgatev** [comment]
I'm okay with the ui5-link change.

File: packages/main/src/Link.js [hunk 1]
```diff
@@ -84,17 +84,21 @@ const metadata = {
...
-		 * @type {boolean}
-		 * @defaultvalue false
+		 * @type {string}
+		 * @defaultvalue "None"
 		 * @public
 		 */
-		wrap: {
-			type: Boolean,
+		wrappingType: {
+			type: String,
```
Comment (fifoosid) [inline packages/main/src/Link.js:100]: The enum that is used for the ```wrappingType``` property of the ```ui5-title``` should be used here as well

**Review by fifoosid — CHANGES_REQUESTED** [review]

File: packages/main/src/Link.js [hunk 2]
```diff
@@ -84,17 +85,20 @@ const metadata = {
...
+		 * Defines how the text of a component will be displayed when there is not enough space.
+		 * Available options are:
+		 * <ul>
+		 * <li><code>None</code> - The text will be truncated with an ellipsis.</li>
+		 * <li><code>Normal</code> - The text will wrap. The words will not be broken based on hyphenation.</li>
+		 * </ul>
 		 *
-		 * @type {boolean}
-		 * @defaultvalue false
+		 * @type {string}
```
Comment (ilhan007) [inline packages/main/src/Link.js:95]: Yes, this becomes a little odd, the type in the JSDoc should be changed
@type {string} -> @type {WrappingType} and both the type and the name of the prop will become "WrappingType":
```js
/**
@type {WrappingType}
*/
 wrappingType: {
			type: WrappingType,
			defaultValue: WrappingType.None,
		},
```

Not sure again, maybe we have to consider naming the enum "Wrapping".

Comment (ilhan007) [inline packages/main/src/Link.js:95]: Small update needed: @ type {string} -> @ type {WrappingType}

**Review by ilhan007 — CHANGES_REQUESTED** [review]

File: packages/main/src/Link.js (hunk 2 above)
Comment (gmkv) [reply packages/main/src/Link.js]: Done

**Review by ilhan007 — APPROVED** [review]

**Pull request merged**